client/src/assets/css/main.css

```diff
@@ -0,0 +1,48 @@
+.text-center{
+    text-align: center;
+}
+
+.text-right{
+    text-align: right;
```

Comment:
That's a helluva css hack right thar. Try and do things more in the spirit of html/css? HTML classes should be descriptive and not an extension of css, else you could just use inline css to be less opaque.